src/frequenz/channels/util/_select.py

```diff
@@ -63,6 +63,9 @@ class Select:
 
     [Receiver][frequenz.channels.Receiver]s also function as `Receiver`.
 
+    When Select is no longer needed, then it should be stopped using
+    `self.stop()` method. This would stop all stored receivers, too.
```

Comment:
it wouldn't stop the receivers,  the receivers will still be usable if people hold on to references to the receivers.

Reply:
Done